lib/user/filestore/client.js

```diff
@@ -48,17 +44,11 @@ class UserFileStoreClient extends Client {
    * @return {object}
    *
    **/
-  constructor (serviceSecret, serviceSlug, userFileStoreUrl, options = {}) {
-    super(
-      serviceSecret,
-      serviceSlug,
-      userFileStoreUrl,
-      UserFileStoreClientError,
-      options
-    )
-
-    this.maxSize = options.maxSize || defaultMaxSize
-    this.expires = options.expires || defaultExpires
+  constructor (serviceSecret, serviceSlug, userFileStoreUrl, encodedPrivateKey) {
+    super(serviceSecret, serviceSlug, userFileStoreUrl, UserFileStoreClientError, { encodedPrivateKey })
```

Comment:
This had an `options` argument with defaulted `maxSize` and `expires` fields. But nothing ever changes those fields

There is no reason why a) we accept an object in this position for the file store client, since we never assign fields _in addition to_ `encodedPrivateKey` anyway, b) we transform every `encodedPrivateKey` in this position to `{ encodedPrivateKey }` since the base class _only ever_ destructs `encodedPrivateKey` back out again (we take a string, embed it in an object, then take out the string again)